src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/DiagnosticDescriptorCreationAnalyzer.cs

```diff
@@ -305,18 +360,62 @@ bool IsCreateHelper(IMethodSymbol method)
                     method.Parameters[0].Type.SpecialType == SpecialType.System_String;
         }
 
-        private static void AnalyzeTitle(IMethodSymbol descriptorCreationMethod, IFieldInitializerOperation creation, Action<Diagnostic> reportDiagnostic)
+        private static void AnalyzeTitle(OperationAnalysisContext operationAnalysisContext, ImmutableArray<IArgumentOperation> creationArguments, IFieldInitializerOperation creation)
+        {
+            IArgumentOperation titleArgument = creationArguments.FirstOrDefault(a => a.Parameter.Name.Equals("title", StringComparison.OrdinalIgnoreCase));
+
+            if (titleArgument != null &&
+                titleArgument.Parameter.Type != null &&
+                titleArgument.Parameter.Type.SpecialType == SpecialType.System_String)
```

Comment:
```suggestion
            if (titleArgument?.Parameter.Type.SpecialType == SpecialType.System_String)
```

Reply:
You are missing a null check for the `Type`, is it on purpose?

Reply:
Yes, parameter type cannot be null.